src/components/controllers/style.module.css

```diff
@@ -34,7 +34,7 @@
 			padding-left: 50px;
 		}
 
-		.full-width {
+		:global(.full-width) {
```

Comment:
Worth calling out that Vite, or whatever Vite's using to process CSS Modules, seemingly has a rather different interpretation of `:global` than other tools. We should probably try to get rid of all usages of `:global` in the future, as it's pretty fiddly.